README.md

```diff
@@ -3,30 +3,88 @@ Styringsværktøj til IT projekter
 
 ## Setup locally
 
+### Preset
+Make sure you have a set of JSON files for testing import Commands.
+
 ### Start Docker containers
 
 ```sh
-docker-compose up -d
-docker-compose exec phpfpm composer install
-docker-compose exec phpfpm bin/console doctrine:migrations:migrate --no-interaction
+docker compose up -d
+docker compose exec phpfpm composer install
+docker compose exec phpfpm bin/console doctrine:migrations:migrate --no-interaction
 ```
 
 ### Create a super admin user
 
 ```sh
-docker-compose exec phpfpm bin/console fos:user:create --super-admin
+docker compose exec phpfpm bin/console fos:user:create --super-admin
 ```
 
 ### Access the site
 
 You should now be able to browse to the application
 
 ```
-open http://$(docker-compose port nginx 80)
+
+open "http://$(docker-compose port nginx 8080)"
+
 ```
 
 ## Import systems and reports
 ```sh
-bin/console itstyr:import:system PATH
-bin/console itstyr:import:report PATH
+docker compose exec phpfpm bin/console itstyr:import:system PATH
+docker compose exec phpfpm bin/console itstyr:import:report PATH
+```
+
+### Flowchart
+
+A helpful flowchart over the Entities, and Joinedtables.
+Ilustrative figures meaning  1 ``` Database = Database ``` 2 ```Black square = Entities``` 3  ```Grey square = relations```
+4 ```Hexagon = Joinedtables``` 5 ```arrow = relation between DB and Entity``` 6 ```bulletin = shows the mapping Entities in Jointables, and JoinCollums```
```

Comment:
Don't use https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks (`` ```…``` ``) for inline code; use https://www.markdownguide.org/basic-syntax/#code.

Reply:
changed